.travis.yml

```diff
@@ -4,8 +4,15 @@
 
 osx_image: xcode9.2
 language: objective-c
-cache: cocoapods
+cache:
+  - cocoapods
+  - bundler
+  - npm
 install:
+  - nvm install 8.9.4
```

Comment:
this is less than ideal but nvm does not play well with travis on this base image so for now this will do